src/cc/torchdistx/fake.cc

```diff
@@ -491,7 +491,9 @@ bool FakeHandler::hasKernelForDispatchKey(DispatchKey key) const noexcept {
 
 void FakeHandler::convertMetaOutputsToFakeTensors() {
   auto fn = [this](Tensor& tensor) {
-    convertToFakeTensor(tensor);
+    if (tensor.is_meta()) {
+      convertToFakeTensor(tensor);
```

Comment:
should the API be called convertMetaTensorToFakeTensor in that case, since it seems to not support non-meta tensors?